Description: Glib 2.31.20 does not allow to include glib/gtypes.h directly any
 more. Instead glib.h needs to be included.
Author: Benjamin Drung <bdrung@debian.org>
Bug: http://bugzilla.audacityteam.org/show_bug.cgi?id=484

--- a/src/AudacityApp.cpp
+++ b/src/AudacityApp.cpp
@@ -406,7 +406,7 @@
  * Happily we can avoid the hack, as we only need some of the headers, not
  * the full GTK headers
  */
-#include <glib/gtypes.h>
+#include <glib.h>
 #include <glib-object.h>
 
 typedef struct _GnomeProgram GnomeProgram;
